### PR TITLE
Minor improvements in TaskService

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/services/data/TaskService.java [hunk 1]
```diff
@@ -811,10 +801,11 @@ public void setShowAutomaticTasks(boolean showAutomaticTasks) {
     public List<TaskDTO> findOpenNotAutomaticTasksWithoutCorrectionForCurrentUser(String sort) throws DataException {
         User user = serviceManager.getUserService().getAuthenticatedUser();
         List<JsonObject> results = findByProcessingStatusUserPriorityAndTypeAutomatic(TaskStatus.INWORK, user.getId(),
-            10, false, sort);
+                10, false, sort);
         return convertJSONObjectsToDTOs(results, false);
     }
 
+
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/services/data/TaskService.java]: Why did you add this empty line?

Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/services/data/TaskService.java]: Mistake :) I will remove it.
